build/config/android/rules.gni

```diff
@@ -4556,6 +4556,13 @@ if (enable_java_templates && is_android) {
 
     _use_scanned_assets = !_ignore_assets && _scanned_files.assets != []
     _has_resources = _scanned_files.resources != []
+    _common_deps = [ ":$_unpack_target_name" ]
```

Comment:
@andrewsavage1 can you suggest what we can do here to avoid mods to upstream ? if we want, lets bracket this with `if()`

Reply:
I recommend cherry picking the actual upstream commit into our repo to avoid conflicts in the future

Reply:
Do you have instructions for it? Or could you do it for this fix?

Reply:
@dahlstrom-g can help out with it, would be good to have a short page explaining how to do it on go/cobalt

Reply:
Agree with Andrew - just cleanly applying the cherrypick is the right thing here, i overlooked the fact that it was upstream. `git am` is probably the right way.

Reply:
If we land this correctly this LGTM now given this is cherry-picked from upstream.